cleverhans/defenses.py

```diff
@@ -0,0 +1,76 @@
+from .loss import Loss
+from .model import Model
+import tensorflow as tf
+
+
+class LossCrossEntropy(Loss):
```

Comment:
Is there a particular reason for defining the three losses `LossCrossEntropy`, `LossMixUp` and `LossFeaturePairing` in `defenses.py` rather than `loss.py`?

Reply:
These are instances used as defense, so I think it makes sense to have the API not depend on them.

Reply:
I'd vote to put them in loss.py too

Reply:
Okay done.